cmd/kubelet/app/server.go

```diff
@@ -861,6 +861,12 @@ func RunKubelet(kubeFlags *options.KubeletFlags, kubeCfg *componentconfig.Kubele
 		}
 	}
 
+	// set up /var/lib/kubelet sharing
+	err = kubeDeps.Mounter.MakeShared(kubeCfg.RootDirectory)
```

Comment:
I hope RunKubelet is the right place to do this startup initialization. Feel free to suggest a better one.